index.js

```diff
@@ -106,6 +106,10 @@ function makeInterceptor (spec) {
 
     return Interceptor.attach(impl, {
       onEnter (args) {
+        if (shouldSkip !== undefined && shouldSkip.call(this)) {
```

Comment:
```suggestion
        if (shouldSkip?.(this)) {
```

Perhaps it's better to pass the `InvocationContext` as a parameter? In hindsight I regret using `this` in `Interceptor`, as it doesn't mix well with arrow functions, and I've also noticed that the trend among Node.js libraries seems to be to pass as an argument instead of using `this`.